app.js

```diff
@@ -0,0 +1,24 @@
+
```

Comment:
Now that we dont use index.js, we will need to change the npm start script to do nodejs app.js